docs/en/Audit-Logging.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: global
+title: Audit Logging
+group: Features
+priority: 99
```

Comment:
Any reason for making the priority so low?

Reply:
No particular reason, just thought it may not be critical.

Reply:
Done.